Backend/hairify/chat/views.py

```diff
@@ -102,20 +102,36 @@ def imageInput(request):
 # {"message":"https://th.bing.com/th/id/OIP.FXBuJ8RmUJkOTSC-qd-M6wHaEK?rs=1&pid=ImgDetMain", "id": 69}
     try:
         id = request.user.id
+        context = request.data['problem']
+        print(context)
```

Comment:
Who is passing the context?

Reply:
So, **context** is just a variable to store the description of the problem that user enters while generating report. It's value is coming as POST request from frontend.